### DISCUSSION TITLE
"Edit Profile" button don't work when on stars list

### DISCUSSION THREAD

**Discussion by fabriceyopa** [community/community#8933] [Projects and Issues]
Hi Teams,

I really enjoy the new feature list, I think there is a small issue, when we can on the stars list, we can edit the profile on the left side.

**Comment by Zopfi** [comment]
Bacon and eggs

**Reply by shenanigans77-jpg** [reply]
Do you see an improvement?

**Comment by Zopfi** [comment]
Eggs and bacon

**Comment by Zopfi** [comment]
Bacon and eggs

**Comment by Zopfi** [comment]
Bacon

**Comment by Zopfi** [comment]
Pancakes

**Comment by Zopfi** [comment]
Heading
